NEWS.md

```diff
@@ -1,3 +1,76 @@
+1.1.0.rc1 (2021-05-13)
+======================
+This is the biggest Hydra release yet.  
+#### Highlights
+- OmegaConf 2.1 supports relative interpolations, nested interpolations, more powerful resolvers and better compatibility with plain Python dict and list.
+- Recursive defaults list: Every config can now have a Defaults List.  
+- Recursive instantiation: When instantiating objects using the `instantiate` API, nested defined objects are instantiated automatically.
+
+There have also been many bug fixes and performance improvements, as well of as some breaking changes - both in Hydra and in OmegaConf.
+Please check the API changes and deprecation sections in this release notes and in the OmegaConf release notes.
+
+### Features
+#### General enhancements
+- Upgrade to OmegaConf 2.1. Please check the [release notes](https://github.com/omry/omegaconf/releases/tag/v2.1.0.rc1) for it ([#1426](https://github.com/facebookresearch/hydra/issues/1426))
+- Support for Python 3.9. ([#1062](https://github.com/facebookresearch/hydra/issues/1062))
+- Improve performance of config composition in a benchmark by 64% ([#1328](https://github.com/facebookresearch/hydra/issues/1328))
+- Allow `@`, `$` and `?` symbols in unquoted values in overrides ([#1074](https://github.com/facebookresearch/hydra/issues/1074), [#1437](https://github.com/facebookresearch/hydra/issues/1437), [#1597](https://github.com/facebookresearch/hydra/issues/1597))
+- Add a --resolve flag that can be used with the --cfg flag to resolve interpolations before the config is printed. ([#1585](https://github.com/facebookresearch/hydra/issues/1585))
+- It is now possible to disable Hydra's logging configuration ([#1130](https://github.com/facebookresearch/hydra/issues/1130))
+- Support for Zsh tab completion ([#347](https://github.com/facebookresearch/hydra/issues/347))
+- Support for force-add of config values via `++key=value` (force-add overrides if the value exists and adds it otherwise) ([#1049](https://github.com/facebookresearch/hydra/issues/1049))
+
+#### Config composition enhancements
+- Support for Defaults List in any config (Recursive relative defaults) ([#1170](https://github.com/facebookresearch/hydra/issues/1170))
+- Composition order of a config with a Defaults List can be specified with the _self_ keyword. ([#326](https://github.com/facebookresearch/hydra/issues/326))
```

Comment:
Rendered as italics otherwise
```suggestion
- Composition order of a config with a Defaults List can be specified with the `_self_` keyword. ([#326](https://github.com/facebookresearch/hydra/issues/326))
```